cmd/gapit/flags.go

```diff
@@ -322,6 +322,7 @@ type (
 		Capture struct {
 			Frames int `help:"only capture the given number of frames. 0 for all"`
 		}
+		RespectAndroidFrameBoundary bool `help:"If true: respect the ANDROID_frame_boundary extension. If false: ignore."`
```

Comment:
Can we rename this to IgnoreFrameBoundaryExtension and change the default value, hint text and behaviour accordingly?

Reply:
Apparently not, but good catch. Added